lib/modules/platform/github/index.spec.ts

```diff
@@ -2440,7 +2464,7 @@ describe('modules/platform/github/index', () => {
     });
     it('returns empty if disabled', async () => {
       // prettier-ignore
-      httpMock.scope(githubApiHost).post('/graphql').reply(200, {data: { repository: {} }} );
+      httpMock.scope(githubApiHost).post('/graphql').reply(200, {data: {repository: {}}});
```

Comment:
Avoid unrelated changes in PRs like this (disable the option in your IDE if necessary)